GOVERNANCE.rst

```diff
@@ -320,31 +331,47 @@ corresponding decision making process is described in more detail below.
    * - Code changes and major documentation changes
      - Lazy consensus
    * - Changes to the API design, hard dependencies, or supported versions
-     - Lazy consensus based on an :ref:`steps`
+     - Lazy consensus, requires a :ref:`steps`
    * - Changes to sktime's governance (this document and the CoC)
-     - Lazy consensus based on an :ref:`steps`
+     - Lazy consensus, requires a :ref:`steps`
    * - Appointment
-     - Voting
+     - Directly starts with voting (stage 2)
 
 .. _stage-1:
 
-Stage 1: lazy consensus with veto right
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Stage 1: lazy consensus
+~~~~~~~~~~~~~~~~~~~~~~~
 
 sktime uses a “consensus seeking” process for making decisions. The
 community tries to find a resolution that has no open objections among
 core developers.
 
--  To accept proposed changes, we require approval by one core developer
-   (lazy consensus) and no rejection by a core developer (veto right).
--  Approvals and rejections can be expressed as +1 and -1 comments,
-   respectively.
--  Core developers are expected to give reasonable time to others to
-   give their opinion on the pull request if they’re not confident
-   others would agree.
--  More important changes that impact the full project require a more
-   detailed analysis and a consensus that is both explicit and informed.
-   These changes require an :ref:`steps`.
+-  Proposed changes should be in the form of GitHub pull requests (PR).
+   Some changes also require a worked out :ref:`steps`, this depends on the type of change, see
+   `decision making process <#Decision-making>`__ above.
+-  For a proposed change to be approved via lazy consensus, it needs to
+   approval by at least one core developer and no rejection by a core developer (veto right).
+   The approval required for this condition must be by a core developer different from a proposer.
+-  For a proposed change to be rejected via lazy consensus, it needs to receive a
+   rejection by at least one core developer, and no acceptance by a core developer.
+-  Approvals must be in the form of a GitHub PR approval. of the PR in question.
+   Rejections can be expressed as -1 comments, or any written comments
+   containing "I formally reject" in the PR, in reference to it.
```

Comment:
```suggestion
   Rejections must be in the form of a GitHub PR rejection on the PR in question.
```

Reply:
there is no such thing as a GitHub PR rejection?